src/main/java/gregtech/api/terminal/TerminalRegistry.java

```diff
@@ -84,29 +85,39 @@ public static void init() {
 
         AppRegistryBuilder.create(new ProspectorApp(0))
                 .battery(GTValues.MV, 1000)
-                .upgrade(MetaItems.COIN_DOGE.getStackForm(10))
-                .upgrade(6, MetaItems.COIN_GOLD_ANCIENT.getStackForm())
+                .upgrade(0, MetaItems.SENSOR_LV.getStackForm(1))
+                .upgrade(1, MetaItems.SENSOR_LV.getStackForm(2))
+                .upgrade(2, MetaItems.SENSOR_MV.getStackForm(1))
+                .upgrade(3, MetaItems.SENSOR_MV.getStackForm(3))
+                .upgrade(4, MetaItems.SENSOR_HV.getStackForm(1))
+                .upgrade(5, MetaItems.SENSOR_HV.getStackForm(3))
+                .upgrade(6, MetaItems.SENSOR_IV.getStackForm(1))
```

Comment:
It actually needs the second upgrade tier to be equal to the LV prospector in size, which is problematic because the third tier upgrade places it equal to the HV prospector